src/main/java/pl/wojciechkostecki/revel/controller/LocalController.java

```diff
@@ -41,7 +41,7 @@ public LocalController(LocalService localService) {
 
     @GetMapping("/search")
     public ResponseEntity<List<Local>> getLocalsByName(@RequestParam(required = false) String name){
```

Comment:
You made the param optional with `requied = false`. We need it to be mandatory, so just remove required = false or change it to true.

Reply:
ok 😄